documentation/docs/documentation.md

```diff
@@ -8,7 +8,7 @@ slug: /
 # Welcome to the Monaco docs
 
 <p>
-Monaco is a monitoring as code (MAC) tool that allows you to create, update and version your monitoring configurations in Dynatrace efficiently and at scale.
+Monaco is a monitoring-as-code (MAC) create, update and version your monitoring configurations in Dynatrace efficiently and at scale.
```

Comment:
```suggestion
Monaco is a "monitoring as code" tool that allows you to create, update and version your monitoring configurations in Dynatrace efficiently and at scale.
```

You've dropped part of that sentence :) 
Also I'd suggest to simply spell our 'monitoring as code' and keep it as individual words, as infrastructure as code usually is